doe_dap_dl/dap.py

```diff
@@ -263,10 +263,15 @@ def setup_two_factor_auth(self, username=None, password=None, authcode=None):
             )
         return valid
 
-    def __renew_cert(self):
+    def renew_cert(self, quiet=False):
         """Renews the certificate"""
         if not self._cert:
-            raise ValueError("No certificate to renew")
+            if not quiet:
+                self.__print("No certificate to renew")
+            return False
+        else:
```

Comment:
You actually don't need an `else` here since you have a `return` statement in the `if` block.